content/en/service_management/workflows/datastore.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Apps Datastore
+disable_toc: false
+further_reading:
+- link: "service_management/app_builder/build"
+  tag: "Documentation"
+  text: "Build Apps"
+- link: "service_management/workflows/build"
+  tag: "Documentation"
+  text: "Build Workflows"
+---
+
+{{< callout url="https://docs.google.com/forms/d/1gVVWLg5P3EXkVXBc4qZtqKp_nqykDsFXCMmh6pbvxx0/edit" btn_hidden="false" header="Try the Preview!">}}
+Datastore is in Preview. Use this form to request access today.
+{{< /callout >}}
+
```

Comment:
Could we add a section about the permissions required? https://datadoghq.atlassian.net/wiki/spaces/APPS/pages/4206395778/How+to+Use+Apps+Datastore#Setup-the-permissions

Reply:
The permissions listed there are what your Datadog account needs to have, is that correct?

Reply:
Added a section.